diff -Nur a/mt76.h b/mt76.h
--- a/mt76.h	2018-06-10 05:13:57.575071000 +0800
+++ b/mt76.h	2018-06-10 05:34:37.243025950 +0800
@@ -26,6 +26,12 @@
 #include <net/mac80211.h>
 #include "util.h"

+//  fix error
+//src/mt76.h:579:40: error: missing binary operator before token "("
+// #if LINUX_VERSION_CODE < KERNEL_VERSION(4,17,0)
+ 
+#include <linux/version.h>
+
 #define MT_TX_RING_SIZE     256
 #define MT_MCU_RING_SIZE    32
 #define MT_RX_BUF_SIZE      2048
