diff -Nur a/mt76x2_eeprom.c b/mt76x2_eeprom.c
--- a/mt76x2_eeprom.c	2018-06-10 05:13:57.579071000 +0800
+++ b/mt76x2_eeprom.c	2018-06-10 05:39:39.639014867 +0800
@@ -18,6 +18,11 @@
 #include "mt76x2.h"
 #include "mt76x2_eeprom.h"
 
+
+// fix  mt76x2_eeprom.c:669:16: error: expected declaration specifiers or ‘...’ before string constant
+#include <linux/module.h>
+
+
 #define EE_FIELD(_name, _value) [MT_EE_##_name] = (_value) | 1
 
 static int
diff -Nur a/usb.c b/usb.c
--- a/usb.c	2018-06-10 05:13:57.579071000 +0800
+++ b/usb.c	2018-06-10 05:40:38.939012694 +0800
@@ -18,6 +18,10 @@
 #include "usb_trace.h"
 #include "dma.h"
 
+// fix  src/usb.c:702:15: error: expected declaration specifiers or ‘...’ before string constant
+// MODULE_AUTHOR("Lorenzo Bianconi <lorenzo.bianconi83@gmail.com>");
+#include <linux/module.h>
+
 #define MT_VEND_REQ_MAX_RETRY	10
 #define MT_VEND_REQ_TOUT_MS	300
 
